web-app/js/portal/cart/DownloadPanel.js

```diff
@@ -134,23 +134,25 @@ Portal.cart.DownloadPanel = Ext.extend(Ext.Panel, {
     },
 
     initButtonPanel: function () {
+
+         this.resetLink = new Ext.ux.Hyperlink({
+            text: OpenLayers.i18n("clearLinkLabel", {text: 'Clear and Reset'}) ,
```

Comment:
The 'Clear and Reset' text should be loaded with i18n as well.

(This happens a few times in this PR, I won't comment them all)